_2017_pages/code_of_conduct.md

```diff
@@ -0,0 +1,189 @@
+
+If you believe someone is violating the Code of Conduct, please report this in a timely
+manner. Code of Conduct violations reduce the value of the community for everyone and we
+take them seriously.
+
+You can make a personal or anonymous report by:
+
+* **By Email**: Send a message to any or all Diversity Chairs, which can be reached at
+  oriol.abril.pla@gmail.com and andorra.alexandre@gmail.com. Event organizers will respond
+  promptly.
+* **On Slack**: Join the [PyMCon Slack](https://pymcon.slack.com/join/shared_invite/zt-fk9efmvb-SWvT01TOnykgzFmLgn1iAw)
+  and contact with the Diversity Chairs, Oriol Abril or Alex Andorra.
+* **Online Form**: A variant of the NumFOCUS Code of Conduct Report is available for
+  PyMCon [here](https://numfocus.typeform.com/to/kMgloi8t) (your name and contact
+  information are not required).
+
+During PyMCon, we recommend contacting via email or Slack so we can take
+immediate action if necessary. In addition to contacting directly with the Diversity
+committee, we also encourages you to make a report using our reporting form.
+
+To file a report anonymously please use the [online form](https://numfocus.typeform.com/to/kMgloi8t).
+
+### What to include in a report
+
+Our ability to address any Code of Conduct breaches in a timely and effective manner
+is impacted by the amount of information you can provide, so our reporting form asks you
+to include as much of the following information as you can:
+
+* Your contact info (so we can get in touch with you if we need to follow up). This will
+  be kept confidential. If you wish to remain anonymous, your information will not be
+  shared beyond the person receiving the initial report.
+* The approximate time and location of the incident (please be as specific as possible)
+* Identifying information (e.g. name, nickname, screen name, physical description) of the
+  individual whose behavior is being reported
+* Description of the behavior (if reporting harassing language, please be specific about
+  the words used), your account of what happened, and any available supporting records
+  (e.g. email, GitHub issue, screenshots, etc.). The online form supports
+  attaching up to 5 files and email and Slack have not limit.
+* Description of the circumstances/context surrounding the incident. Let us know if the
+  incident is ongoing, and/or if this is part of an ongoing pattern of behavior
+* Names and contact info, if possible, of anyone else who witnessed or was involved in
+  this incident. (Did anyone else observe the incident?)
+* Any other relevant information you believe we should have
+
+_The Diversity committee will attempt to gather and write down the above information from
+anyone making a verbal report. Recording the details in writing is exceedingly important
+in order for us to effectively respond to reports. If event staff write down a report
+taken verbally, then the person making the report will be asked to review the written
+report for accuracy._
```

Comment:
Is email+slack+form enough? Or should we try to provide a way of submitting verbal reports such as phone of videocall?

Reply:
This is enough, its the standard at pydata in person conferences, and I believe it matches pydata global as well.

Reply:
Okey, then we have to remove this paragraph